tests/baselines/reference/alwaysStrictNoImplicitUseStrict.errors.txt

```diff
@@ -1,8 +1,8 @@
-error TS5102: Option 'noImplicitUseStrict' has been removed. Please remove it from your configuration.
+error TS5101: Option 'noImplicitUseStrict' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.

```

Comment:
Like Jake said,

```suggestion
error TS5101: Option 'noImplicitUseStrict' is deprecated and has stopped functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
```

would be better. I'll approve, but we should probably update this.